webpack.config.js

```diff
@@ -61,17 +61,34 @@ module.exports = {
                     {
                         loader: "babel-loader",
                         options: {
+                            sourceType: "unambiguous",
                             presets: [
                                 process.env.NODE_ENV === "development"
                                     ? [
                                           "@babel/preset-env",
                                           {
                                               targets: {
                                                   browsers: ["last 2 years"]
-                                              }
+                                              },
+                                              useBuiltIns: "entry",
+                                              corejs: "3"
                                           }
                                       ]
-                                    : "@babel/preset-env"
+                                    : [
+                                          "@babel/preset-env",
+                                          {
+                                              useBuiltIns: "entry",
+                                              corejs: "3"
+                                          }
+                                      ]
+                            ],
+                            plugins: [
+                                [
+                                    "@babel/plugin-transform-runtime",
+                                    {
+                                        regenerator: true
+                                    }
+                                ]

```

Comment:
On which of our products is this based on?

Reply:
uxf_webpack, which is being used by ripe-components-vue.
I included here because it was necessary to solve the `regeneratorRuntime` error that was being thrown by the styleguide

Reply:
It should be part of the host application then. In this case, that would be [storybook](https://github.com/ripe-tech/ripe-sdk-components-vue/blob/master/.storybook/webpack.config.js)